### PR TITLE
Feature/sentinel guide

### PR DESCRIPTION
**Pull request by timabrmsn**
Adds a new Nav bar section called Integration Guides (of which Sentinel is our first/only one). Unfortunately I was unable to figure out how to have sub-pages in the guide, as they lost all formatting. So I put them both on this page, if we add more guides for other products we can solve the layout problem then.

### CODE REVIEW

File: source/integration-guides/azure-sentinel-data-collector.md.erb [hunk 1]
```diff
@@ -0,0 +1,154 @@
+
+### Prepare your environment:
+
+- Install py42 into your Python environment: `python3 -m pip install py42`
+- [Create a Code42 API Client](https://support.code42.com/Incydr/Admin/Code42_console_reference/API_clients) to authenticate py42. The `Alerts Read` permission is required to query alerts.
+- Gather your Azure Log Analytics `Workspace ID` and either `Primary Key` or `Secondary Key` to authenticate requests pushing data into Sentinel.
+
+<aside style="margin: 30px">
```
Comment (timabrmsn) [inline source/integration-guides/azure-sentinel-data-collector.md.erb:8]: Slate doesn't have a "callout" option for markdown content, so we have to use plain html for these :(

File: source/integration-guides/index.html.md.erb [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+Sentinel has two methods of ingesting JSON data as Custom Log sources: the data collector HTTP API, and the Log
+Analytics Linux agent. While the examples below will focus on Incydr Alerts, the patterns can be re-used for almost any
+type of data you can pull from the Code42 API or CLI.
+
+### Send Incydr Alerts to Sentinel with py42 and the Azure Monitor data collector API
+
+This guide outlines how to send Incydr Alerts to Sentinel with a Python script using [py42](https://py42docs.code42.com)
+and the [data collector API](https://learn.microsoft.com/en-us/azure/azure-monitor/logs/data-collector-api).
+
+<details style="margin-left: 30px; margin-right: 60px">
```
Comment (timabrmsn) [inline source/integration-guides/index.html.md.erb:28]: Hiding the content isn't ideal (they should really be separate pages), but this works for now to make the page manageable with multiple guides on the same page.

File: source/integration-guides/azure-sentinel-data-collector.md.erb (hunk 1 above)
Comment (tora-kozic) [reply source/integration-guides/azure-sentinel-data-collector.md.erb:8]: :(

File: source/integration-guides/azure-sentinel-log-analytics.md.erb [hunk 3]
```diff
@@ -0,0 +1,69 @@
+
+### Prepare your environment:
```
Comment (tora-kozic) [inline source/integration-guides/azure-sentinel-log-analytics.md.erb:2]: I haven't looked into what sentinel does much, what is the difference between the tables details in this guide vs. the `data-collector` guide?

File: source/integration-guides/index.html.md.erb (hunk 2 above)
Comment (tora-kozic) [reply source/integration-guides/index.html.md.erb:28]: On my local build of this branch the `Expand guide` text looks like normal paragraph text, could we make it bold and/or a different color maybe to make it look like a think you can click on? 

I agree that we should probably figure out how to make them separate pages eventually though.

File: source/integration-guides/azure-sentinel-data-collector.md.erb [hunk 4]
```diff
@@ -0,0 +1,154 @@
+
+### Prepare your environment:
+
+- Install py42 into your Python environment: `python3 -m pip install py42`
```
Comment (tora-kozic) [inline source/integration-guides/azure-sentinel-data-collector.md.erb:4]: We should probably link the docs and give the command for both user guides (consistency between `install py42` and `install code42cli`)

**Review by tora-kozic — DISMISSED** [review]

**Review by tora-kozic — APPROVED** [review]

**Pull request merged**